solidity/dashboard/src/components/LiquidityRewardCard.jsx

```diff
@@ -115,19 +119,19 @@ const LiquidityRewardCard = ({
           )
         }
       >
-        add more lp tokens
+        {gt(lpBalance, 0) ? "add more lp tokens" : "deposit lp tokens"}
```

Comment:
I believe it should be `gt(wrappedTokenBalance)` because we stake the wrapped token in the `LPRewards` contract.

Reply:
Also, I think if the user doesn't have the wrapped token we should display a link `<Link to="<uniswapLink">deposit lp tokens</Link>` which redirects to the uniswap app.

Reply:
^ so probably should be:

```
gt(wrappedTokenBalance, 0) ? 

<SubmitButton
 className={`liquidity__add-more-tokens btn btn-primary btn-lg w-100`}
 onSubmitAction={(awaitingPromise) =>
  addLpTokens(
   wrappedTokenBalance,
   liquidityPairContractName,
   awaitingPromise
  )
 }
>
 add more lp tokens"
</SubmitButton>
:
<Link to="<uniswapLink" className="btn btn-primary btn-lg w-100>deposit lp tokens</Link>
```

Reply:
ah sorry misunderstood Figma views. Everything is ok! pls ignore the comments above.

We want to display `deposit lp tokens` if the current logged user doesn't have any tokens in the LPRewards pool, so changes looking good. Sorry @michalsmiarowski.